### DISCUSSION TITLE
How to hide the reset-to-default button of the parameter tree?

### DISCUSSION THREAD

**Discussion by wuyuanyi135** [pyqtgraph/pyqtgraph#2678] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/11760870/230572437-e8ad59eb-d7ad-42c3-a3cf-53e1217a580a.png)
Hi, I wonder if it is possible to hide the reset default buttons as shown in the red circle?

**Comment by ntjess** [comment, marked as answer]
The quickest way is to pass `default=None` as an option to each parameter whose button you want to hide

**Reply by wuyuanyi135** [reply]
Works! Thank you
